Jellyfin.Server.Implementations/Activity/ActivityManager.cs

```diff
@@ -50,31 +50,31 @@ public async Task CreateAsync(ActivityLog entry)
 
         /// <inheritdoc/>
         public QueryResult<ActivityLogEntry> GetPagedResult(
-            Func<IQueryable<ActivityLog>, IEnumerable<ActivityLog>> func,
+            Func<IQueryable<ActivityLog>, IQueryable<ActivityLog>> func,
             int? startIndex,
             int? limit)
         {
             using var dbContext = _provider.CreateContext();
 
-            var result = func.Invoke(dbContext.ActivityLogs).AsQueryable();
+            var query = func(dbContext.ActivityLogs).OrderByDescending(entry => entry.DateCreated).AsQueryable();
```

Comment:
why ordering is forced?

Reply:
@mark-monteiro said that was desired behavior, and I double-checked and that is what the behavior was before